articles/cosmos-db/set-throughput.md

```diff
@@ -27,7 +27,7 @@ Setting provisioned throughput on a container is the most frequently used option
 
 The throughput provisioned for a container is evenly distributed among its physical partitions, and assuming a good partition key that distributes the logical partitions evenly among the physical partitions, the throughput is also distributed evenly across all the logical partitions of the container. You cannot selectively specify the throughput for logical partitions. Because one or more logical partitions of a container are hosted by a physical partition, the physical partitions belong exclusively to the container and support the throughput provisioned on the container. 
 
-If the workload running on a logical partition consumes more than the throughput that was allocated to that logical partition, your operations get rate-limited. When rate-limiting occurs, you can either increase the provisioned throughput for the entire container or retry the operations. For more information on partitioning, see [Logical partitions](partition-data.md).
+If the workload running on a physical partition consumes more than the throughput that was allocated to that physical partition, your operations get rate-limited. When rate-limiting occurs, you can either increase the provisioned throughput for the entire container or retry the operations. For more information on partitioning, see [Partitioning and horizontal scaling in Azure Cosmos DB](partition-data.md).
```

Comment:
```suggestion
If the workload running on a logical partition consumes more than the throughput that was allocated to the underlying physical partition, it is possible your operations get rate-limited. When one logical partition has disproportionately more requests compared to other partition key values, this is known as a "hot partition."
When rate-limiting occurs, you can either increase the provisioned throughput for the entire container or retry the operations. You should also ensure you have chosen a partition key that evenly distributes storage and request volume. For more information on partitioning, see [Partitioning and horizontal scaling in Azure Cosmos DB](partition-data.md).
```

Reply:
Thank you for the contribution! I've added some suggested wording to clarify exactly when rate-limiting occurs.

Reply:
Looks crystal clear to me! Thank you for your feedback!